src/shogun/CMakeLists.txt

```diff
@@ -407,55 +400,58 @@ IF(BUNDLE_JSON)
   SET(POSTLINKFLAGS ${POSTLINKFLAGS} ${JSON_LDFLAGS})
 ELSE()
   SHOGUN_DEPENDENCIES(
-    LIBRARY JSON
-    SCOPE PUBLIC
-    CONFIG_FLAG HAVE_JSON
-    VERSION 0.11)
+          LIBRARY JSON
+          SCOPE PUBLIC
+          CONFIG_FLAG HAVE_JSON
+          VERSION 0.11)
 ENDIF()
 
 SHOGUN_DEPENDENCIES(
-  LIBRARY LibXml2
-  SCOPE PUBLIC
-  CONFIG_FLAG HAVE_XML)
+        LIBRARY LibXml2
+        SCOPE PUBLIC
+        CONFIG_FLAG HAVE_XML)
+# RapidJSON
+include(external/RapidJSON)
+SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC ${RAPIDJSON_INCLUDE_DIR})
```

Comment:
@vigsterkr is it ok to have RapidJSON as PUBLIC?

Reply:
i guess it was like this in the other PR, right? i mean currently i include the RapidJSON only in .cpp so private should be fine.... but maybe/probably that caused me troubles with unit tests ...?

Reply:
it was private in the other PR, but swig needs the RapidJSON header files

Reply:
why?